doc/man1/ca.pod

```diff
@@ -134,7 +134,7 @@ The default is PEM.
 
 The password used to encrypt the private key. Since on some
 systems the command line arguments are visible (e.g. Unix with
-the 'ps' utility) this option should be used with caution.
+the L<ps(1)> utility) this option should be used with caution.
```

Comment:
Was the previous discussion about L<> just about not linking to nonexisting OpenSSL functions or was there also discussion about linking to external utilities?